src/screens/login.tsx

```diff
@@ -0,0 +1,144 @@
+import { StatusBar } from 'expo-status-bar';
+import {Alert, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {useState} from "react";
+import { NavigationContainer} from "@react-navigation/native";
+import {createReactNativeMatcher} from "@expo/metro-config/build/transformer/createMatcher";
+
```

Comment:
Can you add "import React from 'react';" along with the other imports? I'm getting errors without it